customer-commons/src/main/java/no/unit/nva/customer/model/VocabularySetting.java

```diff
@@ -0,0 +1,94 @@
+package no.unit.nva.customer.model;
+
+import java.net.URI;
+import java.util.Objects;
+
+public class VocabularySetting {
+
+    private String name;
+    private URI id;
+    private VocabularyStatus status;
+
+    public VocabularySetting() {
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public URI getId() {
+        return id;
+    }
+
+    public void setId(URI id) {
+        this.id = id;
+    }
+
+    public VocabularyStatus getStatus() {
+        return status;
+    }
+
+    public void setStatus(VocabularyStatus status) {
+        this.status = status;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        VocabularySetting that = (VocabularySetting) o;
+        return Objects.equals(name, that.name)
+                && Objects.equals(id, that.id)
+                && status == that.status;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(name, id, status);
+    }
+
+    public static final class Builder {
```

Comment:
I'm growing opinionated here, I know. Not sure I would include a builder. I feel in this case particularly — where the constructed object _has_ to have all three values in order to be valid — a simple three-argument constructor is more appropriate. In general these days, I feel builders are a pattern that at best belongs in external client APIs, but I can see the point if we have god classes (in itself an issue).